apps/comps/components/agents-table/index.tsx

```diff
@@ -388,6 +388,68 @@ export const AgentsTable: React.FC<AgentsTableProps> = ({
               enableSorting: true,
               size: 140,
             },
+            {
```

Comment:
these were part of our comps leaderboard, pre-perps https://github.com/recallnet/js-recall/blob/34165173fad3f74b6260c7161aacd2cbafcee947/apps/comps/components/agents-table/index.tsx#L163C6-L210C9